R/utilities-simulation.R

```diff
@@ -180,12 +180,27 @@ simulateModelParallel <- function(structureSets,
       pathFolder = logFolder,
       logTypes = LogTypes$Info
     )
-
-    subsetSimulationResults <- ospsuite::runSimulations(
-      simulations = simulations
-    )
-
-    simulationResults <- c(simulationResults,subsetSimulationResults)
+    
+    # Catch, save and display any error or warning
+    # from ospsuite::runSimulations
+    subsetSimulationResults <- tryCatch({
+       ospsuite::runSimulations(simulations = simulations)
+    },
+    error = function(e) {
+      logErrorThenStop(message = e, logFolder)
```

Comment:
@Yuri05 @pchelle 

Is the workflow supposed to crash if one of the simulations fails?   
Right now, only a warning is thrown and the tryCatch doesn't actually stop the workflow because it never calls the `error` function here.  Pls see the attached example.
[run_with_v11.zip](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/8358719/run_with_v11.zip)